app/views/devise/registrations/edit.html.erb

```diff
@@ -0,0 +1,42 @@
+<% content_for :page_title, "Edit " + resource_name.to_s.humanize %>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :email, class: "label" %>
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", class: "input" %>
+  </div>
+
+  <% if devise_mapping.confirmable? && resource.pending_reconfirmation? %>
+    <div>Currently waiting confirmation for: <%= resource.unconfirmed_email %></div>
+  <% end %>
+
+  <div class="field">
+    <%= f.label :password, class: "label" %> <i class="is-size-7">(leave blank if you don't want to change it)</i>
+    <%= f.password_field :password, autocomplete: "new-password", class: "input" %>
+    <% if @minimum_password_length %>
+      <em class="is-size-7"><%= @minimum_password_length %> characters minimum</em>
+    <% end %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation, class:"label" %>
+    <%= f.password_field :password_confirmation, autocomplete: "new-password", class: "input" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :current_password, class: "label" %> <i class="is-size-7">(we need your current password to confirm your changes)</i>
+    <%= f.password_field :current_password, autocomplete: "current-password", class: "input" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Update", class: "button is-link m-b-md" %>
+  </div>
+<% end %>
+
+<div>
+<h3 class="m-b-sm">Unhappy?</h3>
```

Comment:
Should we move this or make it harder for users to delete their account? Not sure if that's part of our MVP though.